### PR TITLE
Update welcoming text

### PR DESCRIPTION
**Pull request by Zillode**
Also, 'API' is too complex and 'securing connection' gives a good feeling (and it'll be fully true if we enable TLS):)

### CODE REVIEW

File: src/main/res/values/strings.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
     <string name="send_device_id_to">Send Device ID to</string>
 
     <!-- Text for FoldersFragment and DevicesFragment loading view -->
-    <string name="api_loading">Waiting for API</string>
+    <string name="api_loading">Securing Connection</string>
```
Comment (Nutomic) [inline src/main/res/values/strings.xml]: I don't like this, it's not really what we are doing.

File: src/main/res/values/strings.xml [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
     <string name="app_name">Syncthing</string>
 
-    <string name="app_description">Replace proprietary services with something open, trustworthy and decentralized.</string>
+    <string name="app_description">An open, trustworthy and decentralized file synchronization application.</string>
```
Comment (Nutomic) [inline src/main/res/values/strings.xml]: I took this from [syncthing.net](https://syncthing.net/), do you really think we should have a different text?

File: src/main/res/values/strings.xml [hunk 3]
```diff
@@ -194,14 +194,14 @@ Please install a scanner or enter the device ID manually.</string>
     <string name="web_gui_loading">Waiting for GUI</string>
 
     <!-- Shown instead of web_gui_loading if the key does not exist and has to be created -->
-    <string name="web_gui_creating_key">Generating keys. This may take a while.</string>
+    <string name="web_gui_creating_key">Generating secure keys. This may take up to a few minutes.</string>
```
Comment (Nutomic) [inline src/main/res/values/strings.xml]: "This may take a few minutes" seems more concise.

File: src/main/res/values/strings.xml (hunk 2 above)
Comment (Zillode) [inline src/main/res/values/strings.xml]: Yeah, I think we moved beyond the 'replace' and set our own goals

File: src/main/res/values/strings.xml (hunk 1 above)
Comment (Zillode) [inline src/main/res/values/strings.xml]: Any suggestion? Should be as noob friendly as possible imo.
Loading?

**Comment by Zillode** [comment]
Bumped

File: src/main/res/values/strings.xml (hunk 1 above)
Comment (Nutomic) [inline src/main/res/values/strings.xml]: Yeah loading seems fine.

**Comment by Nutomic** [comment]
So I'd also merge this for 0.6 (as is).

**Comment by Zillode** [comment]
@st-jenkins test this please

File: src/main/res/values/strings.xml [hunk 4]
```diff
@@ -18,7 +18,7 @@
     <string name="send_device_id_to">Send Device ID to</string>
 
     <!-- Text for FoldersFragment and DevicesFragment loading view -->
-    <string name="api_loading">Waiting for API</string>
+    <string name="api_loading">Loading...</string>
```
Comment (Nutomic) [inline src/main/res/values/strings.xml]: Can you use an [Ellipsis](https://en.wikipedia.org/wiki/Ellipsis) here to fix that lint warning?

**Comment by Nutomic** [comment]
Merged in c3d4a1cfdc6a9219577d3479e575583b49dc8024

**Pull request closed without merging**